CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## [v2.24.0](https://github.com/auth0/node-auth0/tree/v2.24.0) (2020-03-05)
+
+**Fixed**
+
+- Fixes test on supportedAlgorithms [\#466](https://github.com/auth0/node-auth0/pull/466) ([davidpatrick](https://github.com/davidpatrick))
+- Add client secret to options for passwordless [\#465](https://github.com/auth0/node-auth0/pull/465) ([davidpatrick](https://github.com/davidpatrick))
```

Comment:
"Minor" is fine. But this should be `CH: Added`. You'd need to re-label it and re-generate this changelog